### PR TITLE
Fixes #12606: Restricted java security policy breaks Rudder (class configured for Cipher(provider: BC)cannot be found)

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/12606

### CODE REVIEW

File: 10_installation/05_requirements/10_jvm_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+[[jvm-requirements]]
+==== JVM Security Policy
+
+Rudder needs `unlimited strength` security policy because it used a variety of advanced hashing and cryptographique
```
Comment (amousset) [inline 10_installation/05_requirements/10_jvm_requirements.txt]: uses

Comment (amousset) [inline 10_installation/05_requirements/10_jvm_requirements.txt]: cryptographic algorithms

File: 10_installation/05_requirements/10_jvm_requirements.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+[[jvm-requirements]]
+==== JVM Security Policy
+
+Rudder needs `unlimited strength` security policy because it used a variety of advanced hashing and cryptographique 
+algorithme only available in that mode. 
+
+Any recent JVM (JDK 8 > 8u161, all JDK 9 and more recent) is configured by default with these policy.
```
Comment (amousset) [inline 10_installation/05_requirements/10_jvm_requirements.txt]: this policy

File: 10_installation/05_requirements/10_jvm_requirements.txt [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+You can check your case by running the following command on your server: 
+
+----
+
+jrunscript -e 'exit (javax.crypto.Cipher.getMaxAllowedKeyLength("RC5") >= 256 ? 0 : 1);'; echo $?
+
+----
+
+If it returns 0, you have the correct policy. In other cases, you will need to change them.
```
Comment (amousset) [inline 10_installation/05_requirements/10_jvm_requirements.txt]: You will need to change it

File: 10_installation/05_requirements/10_jvm_requirements.txt [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+----
+
+If it returns 0, you have the correct policy. In other cases, you will need to change them. 
+
+For that, you can download the  
+http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html[`unlimited strength` policy for JDK 8 here].
+
+
+Then, simply copy the `java.policy` file into `$JAVA_HOME/jre/lib/security/java.policy`.
```
Comment (amousset) [inline 10_installation/05_requirements/10_jvm_requirements.txt]: Missing newline at the end

**Comment by fanf** [comment]
PR rebased

**Pull request merged**